### PR TITLE
UGENE-7406. Crash when execute workflow Read annotation->Write anno…

### PR DESCRIPTION
**Pull request by yalgaer**
…tation with different formats. Part 2: The fix.

### CODE REVIEW

File: src/corelibs/U2Core/src/util/U2FeatureUtils.cpp [hunk 1]
```diff
@@ -94,7 +94,12 @@ QList<U2Feature> U2FeatureUtils::exportAnnotationDataToFeatures(const QList<Shar
 
         // add sub-features
         if (annotation->location->isMultiRegion()) {
-            U2FeatureUtils::addSubFeatures(annotation->location->regions, annotation->location->strand, feature.id, rootFeatureId, dbiRef, os);
+            U2FeatureUtils::createSubFeatures(annotation->location->regions.mid(1),
```
Comment (DmitriiSukhomlinov) [inline src/corelibs/U2Core/src/util/U2FeatureUtils.cpp:97]: Do not get it - why did you remove the first element?

**Review by DmitriiSukhomlinov — CHANGES_REQUESTED** [review]
A question

File: src/corelibs/U2Core/src/util/U2FeatureUtils.cpp (hunk 1 above)
Comment (yalgaer) [reply src/corelibs/U2Core/src/util/U2FeatureUtils.cpp:97]: The first region is always saved as a part of the parent/top-level annotation.

**Review by DmitriiSukhomlinov — APPROVED** [review]
